src/command.cc

```diff
@@ -634,7 +634,16 @@ void Date() {
 }
 
 void Run(TextBox& tbox) {
-  const char* line = tbox.GetRecordedString();
+  const char* raw = tbox.GetRecordedString();
+  char line[128 + 1];
```

Comment:
it may be a good idea to check if the size of `line` buf is bigger than `strlen(raw)+1` to avoid out-of-range write.

Reply:
you can use assert() to verify it. (but it causes a Panic when the assertion is failed, so just using if may be better, but whichever is okay ;)

Reply:
To verify it, I used `TextBox::kSizeOfBuffer` to specify the size of `line`. This makes `TextBox::kSizeOfBuffer` public. Is it ok?